crates/cargo-contract/templates/new/_Cargo.toml

```diff
@@ -5,11 +5,8 @@ authors = ["[your_name] <[your_email]>"]
 edition = "2021"
 
 [dependencies]
-ink_primitives = { version = "4.0.0-alpha.1", default-features = false }
-ink_metadata = { version = "4.0.0-alpha.1", default-features = false, features = ["derive"], optional = true }
-ink_env = { version = "4.0.0-alpha.1", default-features = false }
-ink_storage = { version = "4.0.0-alpha.1", default-features = false }
-ink_lang = { version = "4.0.0-alpha.1", default-features = false }
+# todo: update to crates.io version once released
```

Comment:
We could do a `4.0.0-alpha.2`?

Reply:
I have a follow up to do but yes ASAP

Reply:
https://github.com/paritytech/ink/pull/1400